docs/yarn_classic.md

```diff
@@ -0,0 +1,159 @@
+# Yarn Classic (Yarn v1)
+
+<https://classic.yarnpkg.com/>
+
+* [Overview]
+  * [Supported dependencies types]
+  * [Dealing with Yarn Zero-Installs]
+ * [Fetching dependencies for Yarn Classic projects]
+ * [Handling of yarn-specific config files]
+ * [Setting up offline mirror]
+
+This document outlines the differences between Yarn Classic and Yarn v3 support.
+Please refer to [README] and [Yarn v3 documentation] for
+common aspects of PMs behavior. `Yarn` and `Yarn Classic` will be used interchangeably
+in this document, any other versions of Yarn will be explicitly mentioned.
+
+## Overview
+
+Yarn Classic package manager (PM) relies on Yarn Classic being installed on the system
+where Cachi2 is run. If requested to process a package with Yarn Classic PM it will
+check for yarn version and will refuse to proceed if necessary version is missing.
+Yarn itself is used by Cachi2 under the hood to organize package processing with
+some tweaks to ensure that the packages are prepared to be built in isolation.
+
+The following variables are set for Yarn in the fetch phase:
+
+ * `COREPACK_ENABLE_DOWNLOAD_PROMPT` is set to "0" which prevents
+   Corepack from showing the URL when it needs to download software;
+ * `COREPACK_ENABLE_PROJECT_SPEC` is set to "0" which prevents
+   Corepack from checking if the package manager corresponds to the one
+   defined for the current project;
+ * `YARN_IGNORE_PATH` is set to "true" which ignores any Yarn version specified by a user and
+   uses Corepack's version instead;
+ * `YARN_IGNORE_SCRIPTS`: is set to "true" which prevents execution of any scripts defined in
+   `package.json` or in any dependency;
+ * `YARN_YARN_OFFLINE_MIRROR` is set to point to `deps/yarn-classic` which is relative to
+   output directory and will hold fetched dependencies;
+ * `YARN_YARN_OFFLINE_MIRROR_PRUNING` is set to "false" which prevents Yarn from attempting to
+   ensure that dependencies are up to date.
+
+Once fetch phase is completed Cachi2 will generate an environment file with variables pointing
+to the mirror and instructing Yarn not to prune it:
```

Comment:
> Cachi2 will generate an environment file

True, but it is necessary to run `cachi2 generate-env` ...

Reply:
Good point, will update the wording.